src/js/AutoshareForTwitterPostStatusInfo.js

```diff
@@ -8,6 +8,17 @@ import { useHasFeaturedImage, useAllowTweetImage, useSaveTwitterData } from './h
 
 import { getIconByStatus } from './utils';
 
+// Error message component.
+const ErrorMessage = ( { errorMessage } ) => {
+	return (
+		<span>
+			{ errorMessage }
+			{ ' ' }
+			{ ( errorMessage === 'Failed to tweet; When authenticating requests to the Twitter API v2 endpoints, you must use keys and tokens from a Twitter developer App that is attached to a Project. You can create a project via the developer portal.' ) && <ExternalLink href={ 'https://developer.twitter.com/en/docs/twitter-api/migrate/ready-to-migrate' }>{ __( 'Learn more here.' ) }</ExternalLink> }
```

Comment:
@iamdharmesh, did we leave it intentionally because this message is not translatable?

Reply:
I think it is related to https://github.com/10up/autoshare-for-twitter/pull/229/files/cfcf79d1dbd7e82ef2ac3541f19a98a86cf8419c..cbcfab22b9f84ba313dec5a5c8a1285153639624#r1194775171